doc/nrf/protocols/thread/sed_ssed.rst

```diff
@@ -0,0 +1,63 @@
+.. _thread_sed_ssed:
+
+Sleepy End Device types in Thread
+#################################
+
+Sleepy End Devices (SEDs) and Synchronized Sleepy End Devices (SSEDs) are both :ref:`Minimal Thread Devices (MTDs)<thread_types_mtd>`.
+For more general information about Thread device types, see the documentation for :ref:`thread_device_types`.
+
+Sleepy End Device
+*****************
+
+SEDs are MTDs that sleep most of the time in order to minimize the power consumption.
+They communicate with the Thread network by occasionally polling the parent router for any pending data.
+
+The following Kconfig option is for SED configuration:
+
+* :kconfig:option:`CONFIG_OPENTHREAD_POLL_PERIOD` - Configures the polling period.
+  Higher polling frequency results in lower latency (better responsiveness), but also higher power consumption.
+
+The polling period can also be configured in runtime. See the ``pollperiod`` command in the OpenThread `CLI reference <OpenThread CLI Reference - pollperiod command>`_.
+
+Synchronized Sleepy End Device
+******************************
+
+SSEDs are MTDs that are further optimized for power consumption.
+A Thread SSED is synchronized with its parent router and uses the radio only at scheduled intervals.
+It does this by using the :ref:`thread_ug_supported_features_csl` feature introduced as one of the `Thread 1.2 Base Features`_.
+During those intervals, the device waits for the router to send it any data related to the desired device activity.
+This reduces the network traffic since there is no polling as well as the power consumption since the radio is off most of the time.
```

Comment:
```suggestion
This reduces both the network traffic (since there is no polling) and the power consumption (since the radio is off most of the time).
```